src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.test.tsx

```diff
@@ -107,4 +156,31 @@ describe('<TranscriptsListItemInfo /', () => {
     userEvent.click(proteinLink);
     expect(defaultProps.onProteinLinkClick).toHaveBeenCalled();
   });
+
+  it('display metadata when it is available', () => {
```

Comment:
here and below: `display` -> `displays` (if you start reading from the word "it", the verb should be in 3rd person: it displays)